treeherder/model/models.py

```diff
@@ -1088,7 +1088,7 @@ def to_mozlog_format(self):
         }
 
         # Remove empty values
-        data = {k: v for k, v in data.items() if v}
+        data = {k: v for k, v in data.items() if v is not None}
```

Comment:
This was removing any line that had a line number of `0`.  So this fixes that.  It was messing with some of my tests.